instruqt/avoiding-installation-pitfalls/track.yml

```diff
@@ -1,5 +1,5 @@
 slug: avoiding-installation-pitfalls
-id: amjfqroa5bfk
+id: 
```

Comment:
Should this id be blank?

Reply:
I think the id's are blanked out by the dev/prod workflow